specification/resources/resource-manager/Microsoft.Resources/stable/2024-11-01/resources.json

```diff
@@ -5848,7 +5848,7 @@
```

Comment:
use x-ms-mutability to indicate immutable properties

Reply:
updated